project_templates/stack_package/example_standalone/doc/index.rst

```diff
@@ -0,0 +1,72 @@
+.. py:currentmodule:: lsst.example.standalone
+
+.. _lsst.example.standalone:
+
+#######################
+lsst.example.standalone
+#######################
+
+.. Paragraph that describes what this Python module does and links to related modules and frameworks.
+
+.. .. _lsst.example.standalone-using:
+
+.. Using lsst.example.standalone
+.. =============================
+
+.. toctree linking to topics related to using the module's APIs.
+
+.. .. toctree::
+..    :maxdepth: 1
+
+.. _lsst.example.standalone-contributing:
+
+Contributing
+============
+
+``lsst.example.standalone`` is developed at https://github.com/lsst/example_standalone.
+You can find Jira issues for this module under the `example_standalone <https://jira.lsstcorp.org/issues/?jql=project%20%3D%20DM%20AND%20component%20%3D%20example_standalone>`_ component.
```

Comment:
I don't think T&S is going to use a component for each ts_ package. I *hope* we can convince them to do that. Perhaps talk to Frossie and Andy Clements if you have time?

Reply:
Frossie would be entirely fine with adding components on request. I'd say we leave this in as a default for now and let folks delete this line or adapt it to however they want people to find issues associated with the repo.